### PR TITLE
fix: put autogenerated qa and lqa users under configurable sub-folders

### PR DESCRIPTION
**Pull request by SergiiTao**
related to [LSI-618](https://oat-sa.atlassian.net/browse/LSI-618)

### CODE REVIEW

File: src/Service/Bulk/BulkCreateUsersService.php [hunk 1]
```diff
@@ -33,70 +35,77 @@
 use OAT\SimpleRoster\Repository\Criteria\FindUserCriteria;
 use OAT\SimpleRoster\Repository\UserRepository;
 use OAT\SimpleRoster\Storage\UserGenerator\StorageInterface;
-use Symfony\Component\Security\Core\Encoder\UserPasswordEncoderInterface;
+use Symfony\Component\PasswordHasher\Hasher\UserPasswordHasherInterface;
```
Comment (mianchenia) [inline src/Service/Bulk/BulkCreateUsersService.php:38]: Does it use relevant hash algorithm? In other parts of project the deprecated is in use.

Comment (mianchenia) [reply src/Service/Bulk/BulkCreateUsersService.php:38]: Last time I got issues with login logic with other password encoder.

Comment (SergiiTao) [reply src/Service/Bulk/BulkCreateUsersService.php:38]: `Symfony\Component\Security\Core\Encoder\UserPasswordEncoderInterface` is deprecated
`/api/v1/auth/token` works fine with fresh generated users

**Review by vraginya — APPROVED** [review]

File: src/Service/Bulk/BulkCreateUsersService.php (hunk 1 above)
Comment (mianchenia) [reply src/Service/Bulk/BulkCreateUsersService.php:38]: But what for existed user? Or we don't care?

Comment (SergiiTao) [reply src/Service/Bulk/BulkCreateUsersService.php:38]: have you tried it? :)
I've changed it for encoder only. decoder remains the same, so it obviously should work with existing users.
but yes. it works for both new and old users

**Review by mianchenia — APPROVED** [review]

**Pull request merged**